nav2_graceful_controller/src/parameter_handler.cpp

```diff
@@ -52,21 +54,26 @@ ParameterHandler::ParameterHandler(
     node, plugin_name_ + ".v_linear_max", rclcpp::ParameterValue(0.5));
   declare_parameter_if_not_declared(
     node, plugin_name_ + ".v_angular_max", rclcpp::ParameterValue(1.0));
+  declare_parameter_if_not_declared(
+    node, plugin_name_ + ".v_angular_min_in_place", rclcpp::ParameterValue(0.25));
   declare_parameter_if_not_declared(
     node, plugin_name_ + ".slowdown_radius", rclcpp::ParameterValue(1.5));
   declare_parameter_if_not_declared(
     node, plugin_name_ + ".initial_rotation", rclcpp::ParameterValue(true));
   declare_parameter_if_not_declared(
-    node, plugin_name_ + ".initial_rotation_min_angle", rclcpp::ParameterValue(0.75));
+    node, plugin_name_ + ".initial_rotation_tolerance", rclcpp::ParameterValue(0.25));
   declare_parameter_if_not_declared(
-    node, plugin_name_ + ".final_rotation", rclcpp::ParameterValue(true));
+    node, plugin_name_ + ".prefer_final_rotation", rclcpp::ParameterValue(true));
   declare_parameter_if_not_declared(
     node, plugin_name_ + ".rotation_scaling_factor", rclcpp::ParameterValue(0.5));
   declare_parameter_if_not_declared(
     node, plugin_name_ + ".allow_backward", rclcpp::ParameterValue(false));
+  declare_parameter_if_not_declared(
+    node, plugin_name_ + ".in_place_collision_tolerance", rclcpp::ParameterValue(0.1));
```

Comment:
`in_place_collision_resolution` perhaps? Tolerance feels like a threshold whereas this is a resolution or granularity

Reply:
Ah yeah, that sounds better - I was struggling to come up with the right name for this